crates/driver/Cargo.toml

```diff
@@ -66,6 +66,7 @@ tracing = { workspace = true }
 warp = { workspace = true }
 
 [dev-dependencies]
+ethers = "2.0.9"
```

Comment:
@mfw78 I added my patch to drop `dev-geth` in favor of `anvil` in the `driver` tests. This currently adds `ethers` as a dev dependency to spawn `anvil`.
However, I learned that this doesn't spawn an `anvil` node in the same process and instead simply spawns a new process.
1. That doesn't feel very nice
2. if we actually only want to spawn a process we could also do that using the standard library to not introduce `ethers` as a dependency.
LMK what you think about this change in general.

Reply:
So in the driver we are running the node in the main rust process rather than as a separate thing? Should we try to make this the same across tests?

Reply:
Now each `driver` test spawns a new OS process. This is a bit icky because it requires `anvil` to be in the `$PATH`.
Unfortunately this seems the only way to use `anvil` as it's currently not designed as a separate `crate` that could be run inside the current thread.
Since the `ethers` code is really just spawning a process I decided to get rid of `ethers` (big dependency) again and hand roll that logic.